lib/auth/register.go

```diff
@@ -611,6 +599,41 @@ func pinRegisterClient(params RegisterParams) (*Client, error) {
 	return authClient, nil
 }
 
+func caPathRegisterClient(params RegisterParams) (*Client, error) {
+	tlsConfig := utils.TLSConfig(params.CipherSuites)
+	tlsConfig.Time = params.Clock.Now
+
+	cert, err := readCA(params.CAPath)
+	if err != nil && !trace.IsNotFound(err) {
+		return nil, trace.Wrap(err)
+	}
+
+	// For backwards compatibility we still attempt to use insecureRegisterClient if registration using caPath fails.
```

Comment:
```suggestion
	// For backwards compatibility, we fall back to insecure mode if loading the caPath fails.
	// TODO: At a future date, we should require that a specific path exists to reduce this
	// falling back in an insecure manner - this will be a breaking change.
```

Reply:
Fixed